bindings/c/Cargo.toml

```diff
@@ -36,23 +36,43 @@ cbindgen = "0.26.0"
 [dependencies]
 bytes = "1.4.0"
 # this crate won't be published, we always use the local version
-opendal = { version = ">=0", path = "../../core", features = [
-  "layers-blocking",
-  # These are default features before v0.46. TODO: change to optional features #4313
-  "services-azblob",
-  "services-azdls",
-  "services-cos",
-  "services-fs",
-  "services-gcs",
-  "services-ghac",
-  "services-http",
-  "services-ipmfs",
-  "services-memory",
-  "services-obs",
-  "services-oss",
-  "services-s3",
-  "services-webdav",
-  "services-webhdfs",
-  "services-azfile",
-] }
+opendal = { version = ">=0", path = "../../core", default-features = false, features = ["layers-blocking"] }
 tokio = { version = "1.27", features = ["fs", "macros", "rt-multi-thread"] }
+
+[features]
+default = []
+# Define features that enable corresponding opendal service features
+services-azblob = ["opendal/services-azblob"]
```

Comment:
We can use

```shell
cargo build --features=opendal/services-fs
```

So there is no need to expose all features from bindings side.